code/modules/mob/living/carbon/carbon_defines.dm

```diff
@@ -8,7 +8,7 @@
 
 	var/life_tick = 0      // The amount of life ticks that have processed on this mob.
 
-	var/obj/item/handcuffed = null //Whether or not the mob is handcuffed
+	var/obj/item/restraints/handcuffs/handcuffed = null //Whether or not the mob is handcuffed, restraints/handcuffs/ required for var/cuffed_state
```

Comment:
I also changed this so cuffed_state would work, this is fine right?

Reply:
Looks fine, I'm only ever seeing a type of `obj/item/restraints/handcuffs` being set to this var.